Active Directory integration revamp (#16487)
newrelic/docs-website: src/install/f5/linux/install-linux.mdx

---
componentType: default
headingText: Download the F5 integration
---

1. Change directory to the integrations configuration folder by running:

    ```shell
    cd /etc/newrelic-infra/integrations.d
    ```

2. Copy the sample configuration file by running:

    ```shell
    sudo cp F5-config.yml.sample F5-config.yml
    ```

3. Edit the `F5-config.yml` file. Here's a basic example configuration:

    ```yml
    integrations:
    - name: nri-f5
        env:
        HOSTNAME: localhost
        PORT: 443
        USERNAME: f5_user
        PASSWORD: f5_password
        interval: 15s
        labels:
        environment: production
        inventory_source: config/f5
    ```

> You can find all the config options at the bottom of this doc along with more complex config examples.

Reviewer: link to the anchor subheading?

> You can find all the config options at the bottom of this doc along with more complex config examples.

Reviewer: ```suggestion
You can find all the config options in the [Example configurations section](#examples)  of this doc along with more complex config examples.
```

Reviewer: this could potentially be a bug w/ anchoring w/i a branching doc
